client/src/s3-bucket/s3-client.ts

```diff
@@ -0,0 +1,46 @@
+import { PutObjectCommand, PutObjectCommandInput, S3Client } from '@aws-sdk/client-s3';
+
+const region = 'fra1';
+const endpoint = `https://${region}.digitaloceanspaces.com`;
+
+enum BucketFolders {
+  CertificateOfRegister = 'organization-signup-certificates',
+  UserProfileImages = 'user-profile-images',
+}
+
+const s3Client = new S3Client({
+  forcePathStyle: false, // Configures to use subdomain/virtual calling format.
+  endpoint: endpoint,
+  region,
+  credentials: {
+    accessKeyId: process.env.SPACES_KEY!,
+    secretAccessKey: process.env.SPACES_SECRET!,
+  },
+});
+
+/**
+ * Uploads received file to AWS S3 bucket into a specified folder
+ * @param folder name of the folder to upload the file
+ * @param file the content file to be uploaded
+ * @returns URL of uploaded file
+ */
+const uploadFileToBucket = async (folder: BucketFolders, file: File) => {
+  const params = {
+    Bucket: process.env.S3_BUCKET_ID,
+    Key: `${folder}/${file.name}`,
+    Body: file,
+    ACL: 'private',
```

Comment:
sort by length

Reply:
The parameters are being named according to specific guide: https://docs.digitalocean.com/reference/api/spaces-api/#hello-world-program

Reply:
is javascript and it is an object, and in an object it doesn't matter what order the properties are in